.kitchen.yml

```diff
@@ -3,6 +3,8 @@ driver:
   customize:
     memory: 1024
     cpus: 2
+  ssh:
+    shell: '"/bin/sh"'
```

Comment:
doesn't TK take driver arguments under the platform section as well, so that we can set this only for freebsd?